machine_management/creating_machinesets/creating-machineset-bare-metal.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+[id="creating-machineset-bare-metal"]
+= Creating a compute machine set on bare metal
+include::_attributes/common-attributes.adoc[]
+:context: creating-machineset-bare-metal
+
+toc::[]
+
+You can create a different compute machine set to serve a specific purpose in your {product-title} cluster on bare metal. For example, you might create infrastructure machine sets and related machines so that you can move supporting workloads to the new machines.
+
+include::modules/machine-user-provisioned-limitations.adoc[leveloffset=+1]
+
+include::modules/machineset-yaml-baremetal.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
```

Comment:
I think having this here creates an `.Additional resources` title right before the description. Thats what I'm seeing in the render in the "[Creating a compute machine set section](https://59489--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-bare-metal.html#machineset-creating_creating-machineset-bare-metal)", lmk if thats just me. But I do think this should be removed before merging

Reply:
Thanks Kelly, I removed this. I think it was the remnants of a copy and paste from one of the other sections.

Reply:
Yes! Updates lgtm!